README.md

```diff
@@ -66,21 +66,87 @@ $ yarn install
 $ yarn start
 ```
 
+### Run with `DEV` configuration
+
```

Comment:
I suggest to "start with Why" ... e.g. why do I need to use this, what is this good for.

Reply:
Updated https://github.com/TeskaLabs/asab-webui/pull/89/commits/6b57473be85dc337d434240c105cb5c04d6c6a4a

Reply:
Hmms, I anticipated a documenatation about usage of the fake userinfo.
This is good but very generic.